docs/development/building/Known-Build-Environment-Problems.md

```diff
@@ -48,11 +28,7 @@ $ sudo apt-get install libblocksruntime-dev
 
 ## Compiling Unit Tests fails with errors
 
-Betaflight has been updated to work with clang version 10 (12 on OS X) and will be the official supported (required) version from now on.
-The `Makefile` has been updated to be compatible with this version to run the tests successfully.
-`make test` depends on a specific `clang` compiler. The codebase and tools are updated to work with version 10 on Linux and version 12 on OS X.
-Ubuntu 20.04 LTS and Windows Subsystem for Linux should work with clang version 7, 8 or 10.
-OS X should support clang 7, 10 or 12. Install `clang` on Ubuntu (or Ubuntu using WSL) using:
+Betaflight has been updated to work with clang version 10 (12 on OS X) and will be the official supported (required) version from now on. The `Makefile` has been updated to be compatible with this version to run the tests successfully. `make test` depends on a specific `clang` compiler. The codebase and tools are updated to work with version 10 on Linux and version 12 on OS X. Ubuntu 20.04 LTS and Windows Subsystem for Linux should work with clang version 7, 8 or 10. OS X should support clang 7, 10 or 12. Install `clang` on Ubuntu (or Ubuntu using WSL) using:
```

Comment:
Please update versions to Ubuntu 24.04 and clang 7 - 18 - https://github.com/betaflight/betaflight/pull/14116

IIIRC old clang version is only needed for cygwin.